### PR TITLE
Added GitHub Signup link

### PR DESCRIPTION
**Pull request by adityaraute**
# Pull Request

## Description:

 
Adds GitHub SignUp link to the documentation in 'Create A Local Instance' page.

### CODE REVIEW

File: docs/create-local.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ There are a few steps you will need in order to set up a local instance of Becca
 
 ## Clone the GitHub Repository
 
-Before doing anything else, you have to make sure you have a GitHub Account. If you do not have one, we recommend signing up for free here.
+Before doing anything else, you have to make sure you have a GitHub Account. If you do not have one, we recommend signing up for free [here](https://github.com/signup).
```
Comment (nhcarrigan) [inline docs/create-local.md:7]:
```suggestion
Before doing anything else, you have to make sure you have a GitHub Account. If you do not have one, we recommend [signing up for free here](https://github.com/signup).
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Just a quick a11y tweak.

File: docs/create-local.md (hunk 1 above)
Comment (adityaraute) [reply docs/create-local.md:7]: How are you so quick to reply. Is this automated?

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by adityaraute** [comment]
Okay genuinely though, how did you do this so quick

**Comment by nhcarrigan** [comment]
Activity from our repositories is piped to our discord 🙂

**Pull request merged**